source/dub/commandline.d

```diff
@@ -1372,12 +1372,19 @@ class TestCommand : PackageBuildCommand {
 		args.getopt("f|force", &m_force, [
 			"Forces a recompilation even if the target is up to date"
 		]);
+
 		bool coverage = false;
 		args.getopt("coverage", &coverage, [
 			"Enables code coverage statistics to be generated."
 		]);
 		if (coverage) m_buildType = "unittest-cov";
 
+		bool coverageCTFE = false;
+		args.getopt("coverage-ctfe", &coverageCTFE, [
+			"Enables code coverage (including CTFE) statistics to be generated."
+		]);
+		if (coverageCTFE) m_buildType = "unittest-cov-ctfe";
+
```

Comment:
new undocumented switch?

Reply:
Indeed. Will fix. What about GDC?

Reply:
if there is a GDC switch, add it, otherwise it can be added later.

Reply:
Is there a GDC switch corresponding to dmd/ldc's -cov=ctfe, @ibuclaw?

Reply:
ok then no addition there yet. I would suggest to also remove this extra switch here, which doesn't seem to be a standard switch. (it does the same thing as `--build=unittest-cov-ctfe`)

Reply:
do you want to keep this switch or should we better remove it?